src/ajax/binary.js

```diff
@@ -0,0 +1,17 @@
+import jQuery from "../core.js";
+
+import "../ajax.js";
+
+jQuery.ajaxPrefilter( function( s ) {
+
+	// Binary data needs to be passed to XHR as-is without stringification.
+	if ( typeof s.data !== "string" && !jQuery.isPlainObject( s.data ) ) {
+		s.processData = false;
+	}
+
+	// `Content-Type` for requests with `FormData` bodies needs to be set
+	// by the browser as it needs to append the `boundary` it generated.
+	if ( s.data instanceof window.FormData ) {
+		s.contentType = false;
```

Comment:
Note to self: check if this could be overridden in options if this is set to `false` for all binary data, not just `FormData`.

Reply:
OK, this cannot be changed via settings as of now. :/ I'll need another patch, I think.

Reply:
PR: #5205